shoot/gcp.go

```diff
@@ -118,3 +119,54 @@ func getGCPWorkers(d *schema.ResourceData) []gardener_types.GCPWorker {
 	}
 	return resultWorkers
 }
+
+func updateGCPSpec(d *schema.ResourceData, gcpSpec *gardener_types.GCPCloud) *gardener_types.GCPCloud {
```

Comment:
Usually if a pointer is given as output argument, changes made in the function are available to the caller via the pointer, no need to return it as it only confuses the caller.

Please adjust this function and all other similar ones in this package.

Reply:
now the function doesn't return anything